components/reusable-components/text-input/textInput.module.scss

```diff
@@ -35,6 +36,33 @@
       }
     }
 
+    .newsletterInput {
+      border: 1px solid white;
+      background-color: transparent;
+      color: white !important;
+      height: 3.375rem;
+      -webkit-text-fill-color: #ffff !important;
+      -webkit-text-color: #ffff !important;
+
+      &::placeholder {
+        background-color: transparent !important;
+        color: var(--white-color) !important;
+        font-size: var(--fs-placeholder);
+        font-weight: 300;
+        line-height: var(--fs-placeholder);
+      }
+    }
+
+    .lightNewsletterInput {
+      -webkit-box-shadow: inset 0 0 0 30px #121d36 !important;
+      background-clip: content-box !important;
+    }
+
+    .darkNewsletterInput {
+      -webkit-box-shadow: inset 0 0 0 30px #070b15 !important;
+      background-clip: content-box !important;
```

Comment:
also these colors here, if we don't have them in colors can you please define them there so we can reuse them